docs/DeveloperGuide.md

```diff
@@ -459,10 +459,12 @@ The income or spending will be deleted from its corresponding list using its ind
 
 ## Product scope
 ### Target user profile
-1. prefer desktop apps over other types
+1. Prefer desktop apps over other types
+2. Prefer typing
```

Comment:
maybe add the part on fast typing as well since it was part of the project requirements

Reply:
Fixed!